doc/troubleshooting.md

```diff
@@ -29,6 +29,12 @@ Then simply restart the job server.
 
 Note that the idle-timeout must be higher than request-timeout, or Spray and the job server won't start.
 
+## Timeout getting large job results
+
+If your job returns a large job result, it may exceed Akka's maximum network message frame size, in which case the result is dropped and you may get a network timeout.  Change the following configuration, which defaults to 10 MB:
+
+    akka.remote.netty.tcp.maximum-frame-size = 100 MiB
```

Comment:
what I'm saying is that it says "which defaults to 10 MB:" instead of "which defaults to 10 MiB:"